Sources/AblyChat/ChatAPI.swift

```diff
@@ -3,19 +3,20 @@ import Ably
 internal final class ChatAPI: Sendable {
     private let realtime: RealtimeClient
     private let apiVersion = "/chat/v1"
+    private let apiVersionV2 = "/chat/v2" // TODO: remove v1 after full transition to v2
```

Comment:
I think we're okay to do this as part of this PR

Reply:
You mean replace to v2 everywhere?

Reply:
as in I don't see why we need both apiVersion and apiVersionV2? what do you mean by full transition to V2?

Reply:
"Move to V2 endpoints for history and sending messages." in the doc.

Reply:
It's 404 for occupancy
<img width="1151" alt="Screenshot 2024-11-22 at 01 59 39" src="https://github.com/user-attachments/assets/b1c40dae-a690-48e3-b4f0-87352b432e45">

Reply:
ahh cool, makes sense!

Reply:
So is this that one thing to approval, or renaming?